tests/test_bgp_af.rb

```diff
@@ -840,6 +1137,11 @@ def network_cmd(af, dbg)
     result = af.networks
     assert_equal(should.sort, result.sort,
                  "#{dbg} Test 5. 'Default'")
+    %w(rtmap1, rtmap2, rtmap3, rtmap5, rtmap6, rtmap7,
+       rtmap1_55, rtmap2_55, rtmap3_55,
+       rtmap5_55, rtmap6_55, rtmap7_55).each do |policy|
+      config("no route-policy #{policy}")
+    end if platform == :ios_xr
```

Comment:
I'm not a big fan of putting an if conditional at the end of a block like this - it's not at all obvious. Would much prefer:

```
  return unless platform == :ios_xr
  %w(rtmap1 rtmap2...
     ...).each do |policy|
    ...
  end
end
```